### PR TITLE
Fixed unformat for non standard delimiters

### LINKED ISSUE #328: unformat not working with non localized delimiters
We are using different languages like `en-US` and `de-DE` where delimiters are different according to the language.

When `de-DE` is selected (thousand delimiter is `.`, decimal is `,`) and we call

```
numbro.unformat('180.000')
```

the result is `180` opposed to the expected `180000`. The reason is [this line](https://github.com/BenjaminVanRyseghem/numbro/blob/develop/src/unformatting.js#L194), which is not respecting the delimiters.

```
if (!isNaN(+inputString)) {
        return +inputString;
}
```

I guess this was added for performance reasons!? When I remove it, everything is working as expected.

### PR DESCRIPTION
**Pull request by jkettmann**
Fixes #328

### CODE REVIEW

**Comment by BenjaminVanRyseghem** [comment]
Sorry, I just realised that I left a comment but forgot to submit it

**Comment by BenjaminVanRyseghem** [comment]
are `dist` files tracked?

**Comment by jkettmann** [comment]
Seems like it. I'll remove them

File: tests/src/unformatting-tests.js [hunk 1]
```diff
@@ -89,6 +89,8 @@ describe("unformatting", () => {
 
                 [["(123)", {thousands: ",", decimal: "."}], -123],
 
+                [["12.000", {thousands: ".", decimal: ","}], 12000],
```
[the remark this answers is not in the document]
Comment (jkettmann) [reply tests/src/unformatting-tests.js]: Quick question: Do you want me to move it to the `Regression` section of the file `numbro-tests` or create a new section `Regression` in `unformatting-tests`?

[the remark this answers is not in the document]
Comment (BenjaminVanRyseghem) [reply tests/src/unformatting-tests.js]: creating a new section feels better :smile:

[the remark this answers is not in the document]
Comment (jkettmann) [reply tests/src/unformatting-tests.js]: Ok. Have a look at the changes

**Comment by BenjaminVanRyseghem** [comment]
Could you get ride of `Merge branch 'master' into hotfix/unformat-for-non-standard-delimiters` by rebasing please ?

**Comment by BenjaminVanRyseghem** [comment]
Changes is 👍 

Could you please add a line in CHANGELOG?

I suggest then that you squash all your commits so far, and add a commit where you add yourself to the AUTHOR file.

Thanks :smile:

**Comment by BenjaminVanRyseghem** [comment]
I think it's better to add your name in a separate commit, as if tomorrow we have to revert your change, you'll still be an author.

Now if you don't want to change again, it's not a show stopper :smile:

**Comment by jkettmann** [comment]
That's fine for me. Let's leave it like this

**Comment by BenjaminVanRyseghem** [comment]
done! :smile:

**Comment by BenjaminVanRyseghem** [comment]
in numbro 2.1.2

**Pull request merged**